### DISCUSSION TITLE
Font size for Command Palette

### DISCUSSION THREAD

**Discussion by crustyturd** [sumatrapdfreader/sumatrapdf#2534] [Ideas]
Would it be possible to set font size for Command Palette? It is much too small for me.

**Comment by kjk** [comment]
Not currently.

Are you using latest pre-release? 

I've increased the font for command palette quite a lot at some point. It's  bigger than the size of system font.

**Reply by crustyturd** [reply]
Yes, I am using latest pre-release of 5-5-2022

The font size for command palette is much, much smaller than the menu font.
I am using  4k screen. I know I can set DPI compatibility settings to 'System' in properties of executable, but I prefer to keep DPI compatibility set to 'Application' because it looks much sharper that way.

If it is not possible, I understand. I will live with it. Thanks for the response, and thanks for SumatraPDF.

**Reply by kjk** [reply]
Please try 14543 or later.

I had a small bug in calculating font size. Now it should always be at least 1.4x bigger than the size of system font.

**Reply by crustyturd** [reply]
This is awesome! Thanks!
